docs: DOC-219: Update List tag description and examples

caitlinwheeless opened the pull request:
Update the List tag description and add an example. 

Affects:
- [X] Enterprise docs
- [X] Community docs

caitlinwheeless committed:
docs: DOC-219: Update List tag description and examples

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/9418959371

hlomzik commented on web/libs/editor/src/tags/object/List.jsx:
```diff
@@ -10,9 +10,11 @@ import { parseValue } from "../../utils/data";
- * It's much more lightweight than group of other tags like Text. And you can attach classification to provide additional data about this list.
- * Can be used with `Ranker` tag to rank items or pick relevant items from a list.
+ * The `List` tag is used to display a list of similar items like articles, search results, etc. Task data in the `value` parameter should be an array of objects with `id`, `title`, `body`, and `html` fields.
+ * 
+ * It's much more lightweight to use `List` than to group other tags like Text. Also, you can attach classifications to provide additional data about this list.
+ * 
```
```suggestion
 *
 * It's much more lightweight to use `List` than to group other tags like Text. Also, you can attach classifications to provide additional data about this list.
 *
```
I remove trailing spaces here

hlomzik committed:
Fix linting

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/9420469825
